code/modules/reagents/chemistry_reagents/other.dm

```diff
@@ -34,8 +34,24 @@
 	description = "A ubiquitous chemical substance that is composed of hydrogen and oxygen."
 	reagent_state = LIQUID
 	color = "#0064C8" // rgb: 0, 100, 200
-	custom_metabolism = REAGENTS_METABOLISM * 0.05
-	taste_description = "water"
+	overdose_threshold = REAGENTS_OVERDOSE //I want the 30% chance of the "nauseous message
+	overdose_crit_threshold = REAGENTS_OVERDOSE_CRITICAL * 2 //100u
+	taste_description = "hydration"
+
+/datum/reagent/water/on_mob_life(mob/living/L, metabolism)
+	L.reagents.remove_all_type(/datum/reagent, REM, 0, 1) //slowly washes out chems in you
+	if(volume < 4)
+		L.adjustStaminaLoss(-4*REM)
```

Comment:
This is not weak at all.

Reply:
It actually is really weak.

Turns out that healing 2 stamina for 10 ticks is 20 stamina. Might let you run a little longer in heavy armor, but won't do shit in a fight.

Additionally, you just washed out ~12 ticks/units worth of other medicines, IF you were on them, discouraging pre-medding by "doubling" the metabolism rates of all standard healing chems, and "quadrupling" tramadol metabolism rate.

It's a tradeoff situation.